calico-enterprise/_includes/components/ReqsSys.js

```diff
@@ -29,9 +30,9 @@ function NodeRequirementsEnt(props) {
           <ul>
             {(props.orch === orchestrators.Kubernetes || props.orch === orchestrators.HostProtection) && (
               <>
-                <li>CentOS 8</li>
+                <li>CentOS Stream 8</li>
                 <li>Ubuntu 20.04 and 22.04</li>
-                <li>RHEL 7 and 8</li>
+                <li>RHEL 8 and 9</li>
```

Comment:
I am not sure if RHEL 9 is supported for Calico Enterprise yet.

Reply:
Checked and resolved.